pkgs/applications/office/planner/default.nix

```diff
@@ -10,24 +10,28 @@
 , python
 }:
 
-let version = "20170425";
+let version = "unstable-20180325";
 
 in stdenv.mkDerivation {
   name = "planner-${version}";
 
-  src = fetchgit {
-    url = https://gitlab.gnome.org/GNOME/planner.git;
-    rev = "6a79647e5711b2b8d7435cacc3452e643d2f05e6";
-    sha256 = "18k40s0f665qclrzvkgyfqmvjk0nqdc8aj3m8n4ky85di4qbqlwd";
+  src = fetchFromGitLab {
+    domain = "gitlab.gnome.org";
+    owner = "GNOME";
+    repo = "planner";
+    rev = "2a2bf11d96a7f5d64f05c9053661baa848e47797";
+    sha256 = "1bhh05kkbnhibldc1fc7kv7bwf8aa1vh4q379syqd3jbas8y521g";
   };
 
-  buildInputs = with gnome2; [
+  nativeBuildInputs = [
     pkgconfig
     intltool
     automake111x
     autoconf
     libtool
+  ];
 
+  buildInputs = with gnome2; [
     gnome-common
```

Comment:
`gnome-common`, `gtk-doc` and most likely `scrollkeeper` should go to `nativeBuildInputs` too.

Reply:
Yes ok, it still builds and runs when doing what you suggests.